tests/support/ArrayzyTestTrait.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+trait ArrayzyTestTrait
+{
+  public function simpleArrayProvider()
```

Comment:
Oh, now I see! We have a code duplication! Nice idea, thanks!

P.S. But it would be better to separate *code duplication resolving* and *find method implementation* with a two different PR, then I can merge it quicker.